src/components/AmountInput/AmountInput.vue

```diff
@@ -9,7 +9,9 @@
         class="inputs-container"
     >
         <ErrorTooltip :errors="errors">
-            <input v-model="relativeValue" class="input-style input-size" type="text" />
+            <input v-model="relativeValue" class="input-style amount-input-size" type="text" />
+            <input v-model="TotalAvailableAmount" class="input-style total-amount-input-size" :disabled="true" />
```

Comment:
camelCase for the v-model data. 
`TotalAvailableAmount` ->  `totalAvailableAmount`